src/Components/App/App.js

```diff
@@ -11,6 +11,15 @@ import PageNotFound from '../PageNotFound/PageNotFound'
 import './App.scss'
 
 function App() {
+  if (!('Notification' in window)) {
+    console.log('This browser does not support notifications!')
+    return;
+  }
+
+  Notification.requestPermission(status => {
```

Comment:
Interesting to me that you have to check for notification permission status! Nice work on this though!